programs/hwsim/meta.yaml

```diff
@@ -0,0 +1,10 @@
+short_description: >
```

Comment:
pls use This is a wireless medium simulation tool for Linux, based on the netlink API implemented in the mac80211_hwsim kernel driver.

Reply:
ok